github/projects.go

```diff
@@ -267,6 +267,9 @@ type ProjectCard struct {
 	Note       *string    `json:"note,omitempty"`
 	CreatedAt  *Timestamp `json:"created_at,omitempty"`
 	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
+	Url        *string    `json:"url,omitempty"`
+	Creator    *User      `json:"creator,omitempty"`
+	ColumnID   *int       `json:"column_id,omitempty"`
```

Comment:
I asked in https://github.com/google/go-github/issues/667#issuecomment-313971934, but do you have a source for this field that you can share?